src/main/java/com/buddy/tutorial/controller/NewsApiController.java

```diff
@@ -0,0 +1,39 @@
+package com.buddy.tutorial.controller;
+
+import com.buddy.tutorial.model.ResponseModel;
+import com.buddy.tutorial.model.TopHeadlinesAPIResponse;
+import com.buddy.tutorial.model.TopHeadlinesDetail;
+import com.buddy.tutorial.service.NewsApiService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class NewsApiController {
+
+
+    private static final Integer STATUS_CODE_OK = 200;
```

Comment:
If we need to use the same status codes across multiple files, move it to a common constant file